### PR TITLE
feat: added fliminterface

### PR DESCRIPTION
**Pull request by DeepakMotwani1801**
- added films 
-

### CODE REVIEW

**Review by RoyShubham1012 — COMMENTED** [review]
Good Work!

**Review by zubayer-zidhan — COMMENTED** [review]
looks good.

File: 1.java [hunk 1]
```diff
@@ -0,0 +1,24 @@
+@FunctionalInterface
+interface InterestCalculator{
+    int calculate(int p,int r,int t);
+}
+@FunctionalInterface
```
Comment (RoyShubham1012) [inline 1.java:5]: Add Comments in the code.

**Review by zubayer-zidhan — APPROVED** [review]

**Review by aytidao — DISMISSED** [review]
Working well

**Review by RoyShubham1012 — APPROVED** [review]

File: 1.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+interface GuessHighLow{
+    boolean Guess(int a);
+}
+
+class HigherLowerGame {
+    public  void main(String[] args) {
```
Comment (Tanay74-create) [inline 1.java:11]: missing static  keyword

**Review by aytidao — COMMENTED** [review]
some keywords like static are missing, let's work on this again

File: 1.java (hunk 2 above)
Comment (zubayer-zidhan) [inline 1.java:11]: missing "static" in main method declaration

File: 1.java [hunk 3]
```diff
@@ -0,0 +1,24 @@
+@FunctionalInterface
+interface InterestCalculator{
+    int calculate(int p,int r,int t);
+}
+@FunctionalInterface
+interface GuessHighLow{
+    boolean Guess(int a);
+}
+
+class HigherLowerGame {
```
Comment (Tanay74-create) [inline 1.java:10]: check if it is accessed outside the package

File: 1.java (hunk 1 above)
Comment (zubayer-zidhan) [reply 1.java:5]: the return type of calculate could have been float, because we are printing the float value at the end.

**Pull request merged**